projman_filler/interop_run_stats_parser.py

```diff
@@ -131,7 +131,7 @@ def _get_conversion_results(self) -> list:
                     # These are the same for the lane across all reads
                     total_clusters_pf = row['Reads Pf']
                     # These must be summed
```

Comment:
Please remove this comment, as well as the one that reference the issue on line 124.